mcs/class/System.IO.Compression.FileSystem/Test/System.IO.Compression.FileSystem/ZipTest.cs

```diff
@@ -134,9 +134,10 @@ public void ZipCreateFromEntryChangeTimestamp()
 			{
 				var entry = archive.GetEntry (file);
 				Assert.IsNotNull (entry);
-				Assert.AreEqual(entry.LastWriteTime.Year, date.Year);
-				Assert.AreEqual(entry.LastWriteTime.Month, date.Month);
-				Assert.AreEqual(entry.LastWriteTime.Day, date.Day);
```

Comment:
Are we certain we don't have to check against both, i.e. we are really certain which is UTC and which is local time, or someone else in the system might get them mixed up?

Reply:
`date` is the result of `File.GetLastWriteTimeUtc ()` so yes we can be pretty sure it is in UTC :)